### PR TITLE
Gj computer initialize

### PR DESCRIPTION
**Pull request by grace-joh**
Computer
- created #initialize and tests

Hey @margdaly 
I left a comment in PR #7 (test spec file for Player). My test checks for specific string. Looking forward to your response so we can build our tests (and future tests) similarly!!

*Edit 2/19/23
I will update this spec file according to what we agreed in PR #7

### CODE REVIEW

**Comment by grace-joh** [comment]
Also, I changed the setting for merges to be blocked until PRs are reviewed and approved! FYI so you don't get shocked by the red HAHA

**Review by margdaly — APPROVED** [review]

**Pull request merged**
